### PR TITLE
First Batch of mzSpecLib CV Terms

### PR DESCRIPTION
**Pull request by mobiusklein**
This PR covers the first set of terms introduced from the weekly mzSpecLib call. Most terms are specific to mzSpecLib, but `MS:1003167` in particular introduces a term for ProForma2 sequences and may be of interest in other venues.

### CODE REVIEW

**Review by edeutsch — APPROVED** [review]

**Review by edeutsch — APPROVED** [review]
looks great to me. With the slight totally trivial exception that 3166 - 3172 seem to be missing the comment for their data type relationships that all the other terms seem to have. 
Fine with me to merge with or without that adjustment.

**Comment by mobiusklein** [comment]
That's what I get for attempting to use Bash to solve the problem. I've addressed the missing comments so these terms should be set to go.

**Review by edeutsch — APPROVED** [review]

**Comment by edeutsch** [comment]
waiting for approval from @ypriverol or shall we just go ahead and merge?

**Pull request merged**
